src/main.rs

```diff
@@ -485,6 +506,28 @@ impl AsyncComponent for App {
             gtk::glib::Propagation::Stop
         });
 
+        #[cfg(target_os = "linux")]
+        let tray_img = Icon::from_png_bytes(&TRAY_IMAGE.to_vec()).expect("icon");
```

Comment:
I don't think you need to convert icon to vector first, it is already in byte slice.

Also `.expect()`should be either proof why this never happens or replaced with proper error handling, see other places that already exist in the code. Same comment about other `.expect()` you have here, none of them are proofs, see https://github.com/subspace/subspace/blob/main/CONTRIBUTING.md#code-style

Reply:
done